Makefile

```diff
@@ -14,7 +14,7 @@ requests:
 	@echo "Building Requests"
 	@swift build
 
-run-tests: requests Tests/main.swift $(SPEC_FILES)
+build-tests: requests Tests/main.swift $(SPEC_FILES)
```

Comment:
The result of this rule is creating the `run-tests` file, so it should be named as such.

> A target is usually the name of a file that is generated by a program; examples of targets are executable or object files.

See [makefile documentation](http://www.gnu.org/software/make/manual/make.html#Rule-Introduction).

This rule is saying that, `run-tests` depends on `requests` (phony target because swift-build), and the test source files. If neither of it's dependencies have changed, re-running `run-tests` or `tests` (has a prerequisite of `run-tests`) would result in nothing happening since the file `run-tests` already exists.